assets/index.css

```
@@ -0,0 +1,2989 @@
+	white-space: nowrap;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: normal;
+	font-size: 24px;
+	color: rgba(255,255,255,1);
+}
+.Uptime_Class {
+	left: 729px;
+	top: 795px;
+	position: absolute;
+	overflow: visible;
+	width: 85px;
+	white-space: nowrap;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 25px;
+	color: rgba(244,105,50,1);
+}
+.AddressLeft_Class {
+	display: block;
+	left: 1600px;
+	top: 550px;
+	text-decoration: none;
+	background: none;
+	position: absolute;
+	overflow: visible;
+	width: 64px;
+	white-space: nowrap;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: normal;
+	font-size: 30px;
+	outline: none;
+	color: rgba(244,105,50,1);
+}
+.AddressRight_Class {
+	display: block;
+	left: 1650px;
+	top: 550px;
+	text-decoration: none;
+	background: none;
+	position: absolute;
+	overflow: visible;
+	width: 64px;
+	white-space: nowrap;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: normal;
+	font-size: 30px;
+	outline: none;
+	color: rgba(244,105,50,1);
+}
+
+
+.NextDistributionLabel_Class {
+	left: 672px;
+	top: 543px;
+	position: absolute;
+	overflow: visible;
+	width: 197px;
+	white-space: nowrap;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 25px;
+	color: rgba(244,105,50,1);
+}
+.Version_Class {
+	left: 1622px;
+	top: 178px;
+	position: absolute;
+	overflow: visible;
+	width: 300px;
+	/* white-space: wrap;
+	word-wrap: break-word; */
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 25px;
+	color: rgba(255,255,255,1);
+}
+.Group_4_Class {
+	position: absolute;
+	width: 32px;
+	height: 33px;
+	left: 1087px;
+	top: 179px;
+	overflow: visible;
+}
+.Path_24_Class {
+	fill: rgba(100,177,2,1);
+}
+.Path_24 {
+	overflow: visible;
+	position: absolute;
+	width: 32px;
+	height: 33px;
+	left: 0px;
+	top: 0px;
+	transform: matrix(1,0,0,1,0,0);
+}
+.Path_8_Class {
+	fill: transparent;
+	stroke: rgba(255,255,255,1);
+	stroke-width: 2px;
+	stroke-linejoin: round;
+	stroke-linecap: round;
+	stroke-miterlimit: 4;
+	shape-rendering: auto;
+}
+.Path_8 {
+	overflow: visible;
+	position: absolute;
+	width: 20.35px;
+	height: 13.293px;
+	left: 6.971px;
+	top: 10.481px;
+	transform: matrix(1,0,0,1,0,0);
+}
+.PeerIDLabel_Class {
+	left: 50px;
+	top: 75px;
+	position: absolute;
+	overflow: visible;
+	width: 85px;
+	white-space: nowrap;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 25px;
+	color: rgba(133,133,133,1);
+}
+.PeerID_Class {
+	left: 150px;
+	top: 75px;
+	position: absolute;
+	overflow: visible;
+	width: 557px;
+	white-space: wrap;
+	word-wrap: break-word;
+	text-align: left;
+	font-family: Segoe UI;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 24px;
+	color: rgba(255,255,255,1);
+}
+.EmailLabel_Class {
+	left: 1460px;
```

lets position it from right 
```suggestion
	right: 500px;
```